src/main/java/org/apache/commons/compress/compressors/zstandard/ZstdUtils.java

```diff
@@ -134,4 +161,19 @@ public static void setCacheZstdAvailablity(final boolean doCache) {
     /** Private constructor to prevent instantiation of this utility class. */
     private ZstdUtils() {
     }
+
+    public static long readAndCompressWrite(InputStream content, OutputStream out) throws IOException {
+        byte[] buf = new byte[8192];
```

Comment:
`8192` probably deserves a constant with appropriate name, like `DEFAULT_CHUNK_SIZE`

Reply:
Or reuse `IOUtils.DEFAULT_BUFFER_SIZE`. But, this is likely not needed, see my other comments.